content/applications/finance/accounting/taxes/retention.rst

```diff
@@ -2,68 +2,43 @@
 Withholding taxes
 =================
 
-A withholding tax, also called a retention tax, is a government
-requirement for the payer of a customer invoice to withhold or deduct
-tax from the payment, and pay that tax to the government. In most
-jurisdictions, withholding tax applies to employment income.
+A **withholding tax**, or retention tax, is a government requirement for the payer of a
+customer invoice to withhold or deduct tax from the payment and pay that tax to the government. In
+most jurisdictions, withholding tax applies to employment income.
 
-With normal taxes, the tax is added to the subtotal to give you the
-total to pay. As opposed to normal taxes, withholding taxes are deducted
-from the amount to pay, as the tax will be paid by the customer.
-
-As, an example, in Colombia you may have the following invoice:
-
-.. image:: retention/retention03.png
-   :align: center
-
-In this example, the **company** who sent the invoice owes $20 of taxes to
-the **government** and the **customer** owes $10 of taxes to the **government**.
+Usually, the tax is added to the subtotal to give the total amount to pay. In the case of
+withholding taxes, they are deducted from the amount to pay as the customer pays the tax.
 
 Configuration
 =============
 
-In Odoo, a withholding tax is defined by creating a negative tax. For a
-retention of 10%, you would configure the following tax (accessible
-through :menuselection:`Configuration --> Taxes`):
+In Odoo, a withholding tax is defined by creating a negative tax. To create a retention of 10%, go
+to :menuselection:`Configuration --> Taxes` and in the :guilabel:`Amount` field insert the amount as
+a negative quantity.
 
-.. image:: retention/retention04.png
-   :align: center
+.. image:: retention/negative-amount.png
+   :alt:   negative tax amount in field
 
-In order to make it appear as a retention on the invoice, you should set
-a specific tax group **Retention** on your tax, in the **Advanced Options**
-tab.
+Go to the :menuselection:`Advanced Options` tab and set a specific Retention :guilabel:`Tax Group`.
 
-.. image:: retention/retention02.png
-   :align: center
+.. image:: retention/tax-group.png
+   :alt: tax group for retention tax.
 
-Once the tax is defined, you can use it in your products, sales order or
-invoices.
+Once the tax is defined, it can be used for products, sales orders and invoices.
 
 .. tip::
-    If the retention is a percentage of a regular tax, create a Tax with a
-    **Tax Computation** as a **Tax Group** and set the two taxes in this group
+    If the retention is a percentage of a regular tax, create a :guilabel:`Tax` with a
+    :guilabel:`Tax Computation` as a :guilabel:`Group of Taxes` and set the two taxes in this group
     (normal tax and retention).
```

Comment:
I'm not super sure I understand this. Could we talk about it? In any case, i would change this:
```suggestion
    :guilabel:`Tax Computation` as a :guilabel:`Group of Taxes` and set both the regular and retention taxes in this group.
```